projects/aca-shared/src/lib/components/page-layout/page-layout.component.scss

```diff
@@ -40,6 +40,16 @@
     max-width: 350px;
     width: 350px;
   }
+
+  @media screen and (max-width: 380px) {
+    .aca-page-layout-header {
+      max-height: 16%;
+    }
+
+    .main-content {
+      max-height: 92%;
```

Comment:
I don't get it, why do we need to change max-height for `.main-content`? Isn't it will create a blank space in the bottom of the page?